### PR TITLE
[team_slf]: complete the task_1 

### PR DESCRIPTION
**Pull request by shywon**
### What
- brief:create team
- author:shywon

### What changes are included in this PR?
Add team info & workspace

### Are these changes tested?
Manual test pass

### other information

### CODE REVIEW

File: stl/team_slf/shywon/src/task_1/main.exe
Comment (martinx) [inline stl/team_slf/shywon/src/task_1/main.exe:1]: Please remove the binary.

Comment (shywon) [reply stl/team_slf/shywon/src/task_1/main.exe:1]: ths,i already remove main,exe pls recheck

**Comment by martinx** [comment]
* PR title is not correct. 
* Please submit the team's complete file structure, not an individual part.

**Comment by shywon** [comment]
> * PR title is not correct.
> * Please submit the team's complete file structure, not an individual part.

- PR title is fixed
- The other members did not submit the MR, I will push them to submit it tomorrow

File: stl/team_slf/shywon/doc/task_1/README.md
Comment (Light-City) [inline stl/team_slf/shywon/doc/task_1/README.md:1]: content is empty?

File: stl/team_slf/shywon/README.md
Comment (Light-City) [inline stl/team_slf/shywon/README.md:1]: content is empty?

File: stl/team_slf/README.md
Comment (Light-City) [inline stl/team_slf/README.md:1]: content is empty?

**Comment by Light-City** [comment]
no test、no main 、no readme content, reject.

**Pull request closed without merging**